public/docs/md/charts/en/group-by.md

```diff
@@ -59,4 +59,14 @@ For formulas *with* a group-by, Amplitude ranks the groups by the largest overal
 
 {{partial:admonition type='note'}}
 If group-by pruning occurs with multiple formula terms combined with operators, formulas may take longer to load because Amplitude runs additional queries to make sure that all formula terms are querying the same groups.
-{{/partial:admonition}}
\ No newline at end of file
+{{/partial:admonition}}
+
+## Group By in Experiment
```

Comment:
I am not sure if this article is the right place to put things. Everything in this doc is only for event seg chart but not for example funnel or retention. Should every chart have its own separate doc for group by? 

Also what should the organization of docs look like for features that are in experiment results and e2e experiment? Do we just write it in analytics and then in the appropriate e2e experiment doc just link to that article?